boost-maven/boost-runtimes/runtime-openliberty/src/main/java/org/microshed/boost/runtimes/openliberty/LibertyRuntime.java

```diff
@@ -283,8 +302,8 @@ private Element getRuntimeArtifactElement() {
     }
 
     /**
-     * Invoke the liberty-maven-plugin to package the server into a runnable Liberty
-     * JAR
+     * Invoke the liberty-maven-plugin to package the server into a runnable
+     * Liberty JAR
      */
     private void createUberJar() throws MojoExecutionException {
         executeMojo(getPlugin(), goal("package"),
```

Comment:
Should we skip creating the runnable JAR if we're in dev mode?  Is a running server a good proxy/test for this condition?